### PR TITLE
Add release notes for 5.21.3

### PR DESCRIPTION
**Pull request by cwjohnston**
## Description

Adds release notes for our latest 5.21.x patch.

## Motivation and Context

:shipit:

### CODE REVIEW

File: content/sensu-go/6.0/release-notes.md [hunk 1]
```diff
@@ -127,6 +128,20 @@ See the [supported platforms][165] page for a complete list of Sensu’s support
 With this fix, labels and annotations are always defined (although they may be empty).
 - Fixed the log entry field for the check's name in schedulerd.
 
+## 5.21.3 release notes
+
+**October 14, 2020** &mdash; The latest release of Sensu Go, version 5.21.3, is now available for download.
```
Comment (agoddard) [inline content/sensu-go/6.0/release-notes.md]: @cwjohnston should we say something to qualify this as the latest 5.x release or something?

**Review by agoddard — CHANGES_REQUESTED** [review]
👍 gtg pending any changes re: "latest version of sensu go"

File: content/sensu-go/6.0/release-notes.md (hunk 1 above)
Comment (cwjohnston) [reply content/sensu-go/6.0/release-notes.md]:
```suggestion
**October 14, 2020** &mdash; The latest release of Sensu Go 5, version 5.21.3, is now available for download.
```

File: content/sensu-go/6.1/release-notes.md [hunk 2]
```diff
@@ -178,6 +179,20 @@ See the [supported platforms][165] page for a complete list of Sensu’s support
 With this fix, labels and annotations are always defined (although they may be empty).
 - Fixed the log entry field for the check's name in schedulerd.
 
+## 5.21.3 release notes
+
+**October 14, 2020** &mdash; The latest release of Sensu Go, version 5.21.3, is now available for download.
```
Comment (cwjohnston) [inline content/sensu-go/6.1/release-notes.md]:
```suggestion
**October 14, 2020** &mdash; The latest release of Sensu Go 5, version 5.21.3, is now available for download.
```

**Pull request merged**